aks/application/variables.tf

```diff
@@ -92,8 +92,20 @@ variable "probe_command" {
   description = "Command for the liveness and startup probe"
 }
 
-locals {
-  http_probe_enabled = var.is_web && var.probe_path != null
-  exec_probe_enabled = !var.is_web && length(var.probe_command) != 0
-  probe_enabled      = local.http_probe_enabled || local.exec_probe_enabled
+variable "enable_statuscake" {
+  type        = bool
+  default     = true
```

Comment:
DfE pays for statuscake checks and test environments can be really noisy. Can we default to false instead?

Reply:
Hmm, I wanted us to be explicit here so we're making a considered decision if we're turning off status cake alerts, effectively make the default suitable for production so it's not forgotten. I'd suggest we keep it as defaulting to true and set it to false in other environments, but fair enough if we're worried about services leaving it on for other environments.

On Apply for QTS it's set from the `enable_monitoring` variable: https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/1468/commits/9cdcf0c511c6b10365b6ee684b7f43e291185459

Reply:
Yes I'd rather leave default false. This should be part of a "production checklist" instead

Reply:
No problem, I've amended the commit to default it to `false`.